api/dhcp/v1alpha2/dhcpsubnet_types.go

```diff
@@ -67,6 +69,7 @@ type DHCPAllocated struct {
 // +kubebuilder:printcolumn:name="VRF",type=string,JSONPath=`.spec.vrf`,priority=1
 // +kubebuilder:printcolumn:name="CircuitID",type=string,JSONPath=`.spec.circuitID`,priority=1
 // +kubebuilder:printcolumn:name="Age",type=date,JSONPath=`.metadata.creationTimestamp`,priority=0
+// +kubebuilder:printcolumn:name="DNSServer",type=string,JSONPath=`.spec.dnsserver`,priority=1
```

Comment:
it's better to put it before the Age so it'll be more likely to fit the screen width

Reply:
Done